packages/store/addon/-private/ts-interfaces/identifier.ts

```diff
@@ -1,11 +1,13 @@
 /**
   @module @ember-data/store
 */
-export const IS_IDENTIFIER = Symbol('is-identifier');
+import { symbol } from '../ts-interfaces/utils/symbol';
+
+export const IDENTIFIERS = new WeakMap();
```

Comment:
Nothing in here uses the WeakMap, can you move the WeakMap itself into the location of the utility methods that use it and not export it? (either moving the weakmap there, or moving those helper methods here)

Reply:
Makes sense! Thanks, it's done.